### PR TITLE
make http client faster by disabling the proxy

### PR DESCRIPTION
**Pull request by erwan-joly**

### CODE REVIEW

File: src/Fynance/YahooTicker.cs [hunk 1]
```diff
@@ -80,7 +80,9 @@ public override async Task<FyResult> GetAsync()
 
 		private async Task<HttpResponseMessage> GetResponse(string url)
 		{
-			using (var http = new HttpClient())
+			var handler = new HttpClientHandler();
+			handler.UseProxy = false;
+			using (var http = new HttpClient(handler))
```
Comment (erwan-joly) [inline src/Fynance/YahooTicker.cs:85]: haven't changed it as it shouldn't be in that PR but usually it's bad practice to dispose HttpClient 
It should be reusing the same one all the time.
Might be worth adding http client to a constructor and let the user add their own httpClient if they wish 
and continue to use that way if they don't provide it

Comment (felipeoriani) [reply src/Fynance/YahooTicker.cs:85]: Looks good Erwan! 
I think would be nice to encapsulate the http client instance and add a second constructor which can take the instance of httpClient. Maybe add a method `SetClient(httpClient)`. When it is not defined by the consumer of the component, initialize it with a default instance (as we already have).
What do you think?

Comment (erwan-joly) [reply src/Fynance/YahooTicker.cs:85]: Yes I think that would be a good thing to do. Might be worth to do it in another PR

Comment (felipeoriani) [reply src/Fynance/YahooTicker.cs:85]: Cool, I will close this one, send a new one when you are done and thank you for your contribution!

Comment (erwan-joly) [reply src/Fynance/YahooTicker.cs:85]: I mean this one can be merged in the current state. That's why I believe the other optimisation can be done in another PR

Comment (felipeoriani) [reply src/Fynance/YahooTicker.cs:85]: That's fine, send a new PR with all the modifications around the http client and let's see.

**Pull request closed without merging**